harness/determined/exec/harness.py

```diff
@@ -39,6 +48,9 @@ def main(train_entrypoint: str) -> int:
     # TFKerasTrialController or EstimatorTrialController to add that functionality, so for now we
     # continue with the legacy strategy.
 
+    if isinstance(trial_class, pytorch.PyTorchTrial):
```

Comment:
weird that this is in between the EnvContext comment and the EnvContext, can you move it above the comment?

Reply:
also, mypy will probably catch this but `pytorch` might be `None` here so this isn't quite going to fly.